### ISSUE TITLE
2004 Updates

### ISSUE THREAD

**Issue by EatonZ** [microsoft/Windows-driver-samples#498]
[This](https://github.com/microsoft/Windows-driver-samples/commit/96eb96dfb613e4c745db6bd1f53a92fe7e2290fc) is the last update for some drivers, such as [fastfat](https://github.com/microsoft/Windows-driver-samples/tree/master/filesys/fastfat).

Windows 10 2004 is about to come out. I think it's a good time to freshen up the examples based on the latest 2004 codebase.

**Comment by EatonZ** [comment]
@NeoAdonis I noticed you [published an update](https://github.com/microsoft/Windows-driver-samples/releases/tag/win11-22h2), but there's nothing new? Just an updated readme file - was there a glitch uploading all the rest of the files?

**Comment by v-junyli** [comment]
We are closing out this issue that unfortunately have not been addressed for a long time.  Our apologies for not attending to the issue in a timely manner.  If you have data that issue remains and it is still important, then feel free to reopen.

**Comment by EatonZ** [comment]
@v-junyli Indeed, no one ever commented on this, but the question remains valid. Are you able to tag the right person to get an answer?

**Comment by v-junyli** [comment]
@EatonZ sure I reopened the issue. But I think this version of WDK is too old that is no longer supported.

@prashantchahar can you comment?

**Comment by EatonZ** [comment]
@prashantchahar
